src/templates/layouts/staffProfile/dataService.test.tsx

```diff
@@ -25,4 +25,28 @@ describe('transformStaffProfileData', () => {
 
     expect(result).toEqual(expectedStaffProfile)
   })
+
+  it('should return transformed staff profile with null path', () => {
```

Comment:
thanks for writing this one but we should actually remove the dataService altogether. it was an early AP approach before we landed on the current way of doing queries + formatters.